packages/editor/src/components/post-template/classic-theme.js

```diff
@@ -172,17 +186,14 @@ function PostTemplateDropdownContent( { onClose } ) {
 										{
 											label: __( 'Go back' ),
 											onClick: () =>
-												setRenderingMode(
-													getEditorSettings()
-														.defaultRenderingMode
-												),
+												getEditorSettings().goBack(),
 										},
 									],
 								}
 							);
 						} }
 					>
-						{ __( 'Edit template' ) }
+						{ __( 'Edit templates' ) }
```

Comment:
Why the change of label?

Reply:
Hmm, that is weird, where did that s come from - gone now.